src/Notifier.js

```diff
@@ -244,16 +254,26 @@ const Notifier = {
         if (ev.sender && ev.sender.userId === MatrixClientPeg.get().credentials.userId) return;
         if (data.timeline.getTimelineSet() !== room.getUnfilteredTimelineSet()) return;
 
-        const actions = MatrixClientPeg.get().getPushActionsForEvent(ev);
-        if (actions && actions.notify) {
-            if (this.isEnabled()) {
-                this._displayPopupNotification(ev, room);
-            }
-            if (actions.tweaks.sound && this.isAudioEnabled()) {
-                PlatformPeg.get().loudNotification(ev, room);
-                this._playAudioNotification(ev, room);
+        // If it's an encrypted event and the type is still 'm.room.encrypted',
+        // it hasn't yet been decrypted, so wait until it is.
+        if (ev.isEncrypted() && ev.getType() == 'm.room.encrypted') {
```

Comment:
You probably want the same as https://github.com/matrix-org/matrix-js-sdk/pull/530/commits/ec5f9a28920a859ec42c990fb1769e5f0860ab56#diff-cf27c1d543e886c89cd9ac8b8aeaf05bR3073.